backend/utils/threads_utils.py

```diff
@@ -56,8 +56,40 @@ async def add_message_to_thread_list(org_id, room_id, message_id, request: Threa
     thread_message["created_at"] = str(datetime.utcnow())
 
     message["threads"].insert(0, thread_message)
-   
+
     response = await update_message(org_id, message_id, message)
-    
+
     return response, thread_message
 
+
+async def update_thread_message(
+    org_id: str, message_id: str, thread_id: str, payload: dict[str, Any], message: dict[str, Any]
```

Comment:
Do we still need **message** when **message_id** is already provided?
5 params, This is starting to be a lot.